sdk/src/mcm_dp.c

```diff
@@ -145,6 +145,10 @@ mcm_conn_context* mcm_create_connection(mcm_conn_param* param)
         parse_memif_param(param, &(memif_param.socket_args), &(memif_param.conn_args));
         /* Connect memif connection. */
         conn_ctx = mcm_create_connection_memif(param, &memif_param);
+        if (!conn_ctx) {
+            log_error("Failed to create memif connection.");
+            return NULL;
+        }
```

Comment:
Out of scope - also should be moved as separate PR probably.

Reply:
This should stay with this PR since it fixes a segfault occurring when trying to run FFmpeg with the MCM plugin with the memif protocol set.